### ISSUE TITLE
jormungandr v0.8.0-rc7 got stuck on synchronisation

### ISSUE THREAD

**Issue by congle-ibl** [input-output-hk/jormungandr#1284]
Hi all,

I have run incentivized testnet node with version v0.8.0-rc7 but it got stuck.
Message log on fullnode:
```
1|start-incentivized-testnet  | Dec 05 17:03:07.680 WARN blockchain is not moving up, the last block was 1980187 seconds ago, task: stuck_notifier
1|start-incentivized-testnet  | Dec 05 17:04:07.680 WARN blockchain is not moving up, the last block was 1980247 seconds ago, task:
1|start-incentivized-testnet  |  stuck_notifier
```

And I got nothing for a long time:
```

version | "jormungandr 0.8.0-rc7-ab32b2cd+"
-- | --
state | "Running"
blockRecvCnt | 0
lastBlockDate | "0.0"
lastBlockFees | 0
lastBlockHash | "0f9d564199ad7f71af3daaff4b6997cb7f2e3d7c422fa29097f5d6a018c440d1"
lastBlockHeight | "0"
lastBlockSum | 0
lastBlockTime | null
lastBlockTx | 1
txRecvCnt | 0
uptime | 9019
```

Here is my command:
```
jormungandr --config /home/lecong/workspace/ADA/data_incentivized_testnet/config.yaml --genesis-block-hash '0f9d564199ad7f71af3daaff4b6997cb7f2e3d7c422fa29097f5d6a018c440d1'
```

And configuration at config.yaml:
```
{
  "storage": "/home/lecong/workspace/ADA/data_incentivized_testnet/",
  "p2p": {
    "topics_of_interest": {
      "blocks": "normal",
      "messages": "low"
    },
    "trusted_peers": [
	{
	"address": "/ip4/3.124.55.91/tcp/3000",
	"id": "d5cb4aa6e28a4b4951fb3121365da45466a2a31c880b44a5"
	},
	{
	"address": "/ip4/54.183.206.246/tcp/3000",
	"id": "67282e0bf1dcdf5e4559c85b057de122ba7a248f16ac5226"
	},
	{
	"address": "/ip4/52.8.126.235/tcp/3000",
	"id": "75534bbbb7dddfeee3783ae5e7983c28349209138351c278"
	},
	{
	"address": "/ip4/3.113.195.95/tcp/3000",
	"id": "617ff3920bac9e90faec3adc5d32c2937cbb6b8528b1b22d"
	},
	{
	"address": "/ip4/18.176.14.159/tcp/3000",
	"id": "55f82104122f4d5101ae549935943071c8c646f1bd62cd0b"
	},
	{
	"address": "/ip4/3.124.166.70/tcp/3000",
	"id": "96a84a81a74715c7fb29d906ddff32b2874aac4d14acc7db"
	},
	{
	"address": "/ip4/3.125.15.153/tcp/3000",
	"id": "97fd215d7e294b5943960056af1e1a0316c3593a4aa3cb76"
	},
	{
	"address": "/ip4/3.124.103.156/tcp/3000",
	"id": "173ca3b1198816989c585785a8c912fea3c5ec14895eaa6b"
	}
    ]
  },
  "rest": {
    "listen": "0.0.0.0:8443"
  }
}
```
Should I need to change trusted peers or do something else to fix it? Thank you so much for your support.

**Comment by mmahut** [comment]
Nightly network is down at the moment.

**Comment by congle-ibl** [comment]
Hi @mmahut ,

I'm running incentivized network with this genesis block hash `0f9d564199ad7f71af3daaff4b6997cb7f2e3d7c422fa29097f5d6a018c440d1`. Is it down too? Thank you bro.

**Comment by NicolasDP** [comment]
I believe this is fixed with #1290 and #1292 . It will be included in the next release
